src/components/merge_results/fields/CompareCategory.tsx

```diff
@@ -0,0 +1,88 @@
+import WarningIcon from '@mui/icons-material/Warning';
+import { Typography } from '@mui/material';
+import { useFormContext } from 'react-hook-form';
+import { Trans, useTranslation } from 'react-i18next';
+import { PublicationType } from '../../../types/publicationFieldNames';
+import { PublicationInstanceType, Registration } from '../../../types/registration.types';
+import { StyledInfoBanner } from '../../styled/Wrappers';
+import { CompareFields } from './CompareFields';
+import { SourceValue } from './SourceValue';
+
+interface CompareCategoryProps {
+  sourceInstanceType: PublicationInstanceType | '';
+  targetInstanceType: PublicationInstanceType | '';
+  sourceMainType: PublicationType | '';
+  targetMainType: PublicationType | '';
+}
+
+export const CompareCategory = ({
+  sourceInstanceType,
+  targetInstanceType,
+  sourceMainType,
+  targetMainType,
+}: CompareCategoryProps) => {
+  const { t } = useTranslation();
+  const { formState, setValue } = useFormContext<Registration>();
+
+  const targetInitialInstanceType =
+    formState.defaultValues?.entityDescription?.reference?.publicationInstance?.type ?? '';
+  const isSameMainCategory = sourceMainType === targetMainType;
+
+  return (
+    <>
+      {!isSameMainCategory && (
+        <StyledInfoBanner sx={{ gridColumn: '1/-1', display: 'flex', alignItems: 'center', gap: '1rem' }}>
+          <WarningIcon />
+          <div>
+            <Trans
+              i18nKey="cannot_copy_category_from_another_context_type"
+              components={{ p: <Typography color="inherit" /> }}
+            />
+          </div>
+        </StyledInfoBanner>
+      )}
+
+      <CompareFields
+        sourceContent={
+          <SourceValue
+            label={t('common.category')}
+            value={
+              <>
+                {sourceInstanceType && t(`registration.publication_types.${sourceInstanceType}`)}
+                {!isSameMainCategory && sourceMainType && <MainTypeContent type={sourceMainType} />}
+              </>
+            }
+          />
+        }
+        targetContent={
+          <SourceValue
+            label={t('common.category')}
+            value={
+              <>
+                {targetInstanceType && t(`registration.publication_types.${targetInstanceType}`)}
+                {!isSameMainCategory && targetMainType && <MainTypeContent type={targetMainType} />}
+              </>
+            }
+          />
+        }
+        isMatching={sourceInstanceType === targetInstanceType}
+        isChanged={targetInitialInstanceType !== targetInstanceType}
+        onCopyValue={
+          sourceInstanceType && isSameMainCategory
+            ? () => setValue('entityDescription.reference.publicationInstance.type', sourceInstanceType)
+            : undefined
+        }
+        onResetValue={() => setValue('entityDescription.reference.publicationInstance.type', targetInitialInstanceType)}
```

Comment:
Vil bruke `resetField()` her som for andre plasser, men for en eller annen grunn virker ikke det. Virker å resette `entityDescription.reference.publicationInstance` men tror det kan gjøre mer skade igjen, så da er foreløpig konklusjon å krype til korset med `setValue()` :(

```suggestion
        onResetValue={() => resetField('entityDescription.reference.publicationInstance.type')}
```